docs/pages/push-notifications/faq.mdx

```diff
@@ -87,7 +87,7 @@ curl -H "Content-Type: application/json" -X POST "https://exp.host/--/api/v2/pus
 
 ### Notifications work in development, but not in release mode
 
-This indicates that you have either misconfigured your credentials or didn't configure them at all in your production app. Expo Go uses Expo's credentials, which allows working on notifications in development.
+This indicates that you have either misconfigured your credentials or didn't configure them at all in your production app. In SDK 53 and later, Expo Go does not support push notifications functionality, so to test push you should use a [development build](/develop/development-builds/introduction/). In SDK 52 and earlier, Expo Go used Expo's credentials, which allowed push notifications to work in development without setting up your own credentials.
```

Comment:
Local notifications (in-app notifications) are still available in Expo Go. So it would be more accurate to say that only "remote notifications" are not available.

Reply:
@kadikraman, the section is a push notifications FAQ, and the page is also about push notifications troubleshooting.: https://docs.expo.dev/push-notifications/faq/#push-notifications-troubleshooting. I don't think we need to call local notifications explicitly in this section. Otherwise, it will be verbose and can create confusion as to why/some FAQ items call out local notifications in this doc while others don't.